lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -614,6 +614,11 @@ def test_grades_csv(self):
         )
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            response.__getitem__('content-disposition'),
```

Comment:
Cool. __getitem__ is the method variant of the foo['x'] access pattern. Didn't know if we were doing weird things. :)  Thanks!